src/docs/development/tools/devtools/memory.md

```diff
@@ -81,24 +132,45 @@ See [Allocation tab](#allocation-tab).
   work can be computed and optimized ahead of time, while in [debug mode][]
   that same work might have to be computed at runtime,
   requiring more information.
+
+  对你正在运行的应用程序进行性能分析时，请使用 **[profile 模式][profile mode]**。
+  除非你的应用程序以 profile 模式运行，否则不能看做 release 版本的内存性能表现。
+  一般来说，在 debug、release 和 profile 模式之间，相对而言，内存的使用是相当准确的。
+  因为 profile 模式创建了 service isolate 来配置应用程序，可能会显示更高的绝对内存使用率。
```

Comment:
测量模式？这个有统一的叫法吧~ 或者我们定一个？

Reply:
有统一的叫法：C/S中 profiling 意为「性能分析」，所以 profile mode 应为「性能模式」

Reply:
ref: https://zh.wikipedia.org/wiki/%E6%80%A7%E8%83%BD%E5%88%86%E6%9E%90

Reply:
「性能模式」会不会让人联想成 「比 relesase 版本性能更高」，感觉生活中用手机时打开性能模式时，给人感觉性能会比平时更强劲，在这个场景下是不是「分析/测量」一类的词贴切点。

Reply:
你的联想其实就是对的，性能模式就是要比生产模式少一些限制和加载，所以一般用性能模式来做分析，能够排除一些生产上的问题

Reply:
> Profile mode might show higher absolute memory usage
  because a service isolate is created to profile your application.
嗯嗯 profile 比 debug 性能更好，但是它比 release 要差些，我是担心叫「性能模式」的话，别人会不会理解为它的性能比 release 更好 😟。

Reply:
> Profile mode might show higher absolute memory usage
  because a service isolate is created to profile your application.
嗯嗯 profile 比 debug 性能更好，但是它比 release 要差些，我是担心叫「性能模式」的话，别人会不会理解为它的性能比 release 更好 😟。

Reply:
所以不译我认为也是可以的，cc @Vadaski

Reply:
哈哈，赞同，大家可以看看